src/main/java/com/hedvig/productPricing/service/serviceIntegration/memberService/MemberServiceImpl.java

```diff
@@ -43,4 +44,13 @@ public String getPickedLocale(String memberId) {
       throw e;
     }
   }
+
+  @Override
+  public void notifyContractsCreated(String memberId) {
```

Comment:
```suggestion
  public void notifyContractCreated(String memberId) {
```

Its only one per time